x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -1215,7 +1215,7 @@ response_json using exportId as '2212':
 ----
 This behaviour of targeted pattern replacement in the url helps solve various use cases.
 
-NOTE: Replace patterns should not contain commas `","` in their definition.
+NOTE: Replace patterns must not contain commas `","` or `regular expressions` in their definition.
```

Comment:
I think the added clause here is confusing. Better to not raise this here since it makes the user think that REs are somehow involved and they are no. Calling the pattern in the text above a "fixed pattern", potentially with a final sentence along the lines of "String replacement patterns are matched by the \`replace_with\` processor with exact string matching."

Reply:
updated the docs accordingly.